package.json

```diff
@@ -115,8 +115,12 @@
   "pnpm": {
     "overrides": {
       "ws@>=8.0.0 <8.17.1": ">=8.17.1",
-      "micromatch@<4.0.8": ">=4.0.8",
-      "path-to-regexp@>=0.2.0 <8.0.0": ">=8.0.0"
+      "micromatch@<4.0.8": ">=4.0.8"
+    },
+    "auditConfig": {
+      "ignoreGhsas": [
+        "GHSA-9wv6-86v2-598j"
```

Comment:
Thanks for using this field even though there is a CVE! 🙂

Reply:
I couldn't resist ;)